doc/latex.rst

```diff
@@ -623,76 +621,25 @@ uses keys described later in :ref:`additionalcss`.
-         % pre_border-width is 5.1.0 alias for verbatimborder
-         pre_border-width=2pt,
-         pre_border-right-width=8pt,
-         % pre_padding is a 5.1.0 alias for verbatimsep
-         pre_padding=5pt,
-         % Rounded boxes are new at 5.1.0
-         pre_border-radius=5pt,
-         % TeXcolor reminds that syntax must be as for LaTeX \definecolor
-         pre_background-TeXcolor={named}{OldLace},
-         % ... and since 5.3.0 also xcolor \colorlet syntax is accepted and we
-         %     can thus drop the {named}{...} thing if xcolor is available!
-         pre_border-TeXcolor=Gold,
-         % ... and even take more advantage of xcolor syntax:
-         pre_border-TeXcolor=Gold!90,
-         % add a shadow to code-blocks
-         pre_box-shadow=6pt 6pt,
-         pre_box-shadow-TeXcolor=gray!20,
-         %
-         % This 5.1.0 CSS-named option is alias for warningborder
-         div.warning_border-width=3pt,
-         % Prior to 5.1.0, padding for admonitions was not customizable
-         div.warning_padding=6pt,
-         div.warning_padding-right=18pt,
-         div.warning_padding-bottom=18pt,
-         % Assume xcolor has been loaded with its svgnames option
-         div.warning_border-TeXcolor=DarkCyan,
-         div.warning_background-TeXcolor=LightCyan,
-         % This one is the only option with space separated input:
-         div.warning_box-shadow=-12pt -12pt inset,
-         div.warning_box-shadow-TeXcolor=Cyan,
-         %
-         % The 5.1.0 new name would be div.attention_border-width
-         attentionborder=3pt,
-         % The 5.1.0 name here would be div.attention_border-TeXcolor
-         attentionBorderColor=Crimson,
-         % The 5.1.0 name would be div.attention_background-TeXcolor
-         attentionBgColor=FloralWhite,
-         %
-         % For note/hint/important/tip, the CSS syntax was added at 6.2.0
-         % Legacy syntax still works
-         noteborder=1pt,
-         noteBorderColor=Olive,
-         % But setting a background color via the new noteBgColor means that
-         % it will be rendered using the same interface as warning type
-         noteBgColor=Olive!10,
-         % We can customize separately the four border-widths, and mimic
-         % the legacy "light" rendering, but now with a background color:
-         % div.note_border-left-width=0pt,
-         % div.note_border-right-width=0pt,
-         % Let's rather for variety use lateral borders:
-         div.note_border-top-width=0pt,
-         div.note_border-bottom-width=0pt,
-         %
-         % As long as only border width and border color are set, *and* using
-         % for this the old interface, the rendering will be the "light" one
-         hintBorderColor=LightCoral,
-         % but if we had used div.hint_border-TeXcolor or *any* CSS-named
-         % option we would have triggered the more complex "heavybox" code.
+         ... more key=value in LaTeX syntax ...
```

Comment:
I have removed all of this because I am afraid it may scare away the reader.  But it did give examples of use.